### PR TITLE
Use fullwidth alert component in Banner

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

This is meant to address an a11y issue with the `<AlertBox>` component @timwright12 mentioned in https://github.com/department-of-veterans-affairs/component-library/pull/179. In an effort to further the migration away from React components and towards Web Components, the `<Banner>` will use `<va-alert>` instead of its deprecated React counterpart.

If this gets merged, we should be able to delete [this file](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/a50f7304cfbb9824b75fd8af215769f1cf595c67/packages/formation/sass/modules/_m-emergency-banner.scss#L1).

## Testing done

Storybook :eyes: 

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -99,12 +99,13 @@
     "sass-loader": "^8.0.2",
     "sinon": "^9.2.2",
     "uswds": "1.6.10",
-    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-v0.9.0"
+    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-alert-full"
```
Comment (bkjohnson) [inline package.json:102]: This PR shouldn't get merged until we have a real release to use.

File: src/components/Banner/Banner.jsx [hunk 2]
```diff
@@ -1,9 +1,8 @@
 // Node modules.
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
 // Relative imports.
-import AlertBox from '../AlertBox/AlertBox';
+import { VaAlert } from 'web-components/react-bindings';
```
Comment (bkjohnson) [inline src/components/Banner/Banner.jsx:5]: We're using the React binding so that we can use the `onCloseAlert` event handler prop.

**Review by timwright12 — APPROVED** [review]

File: src/components/Banner/Banner.jsx [hunk 3]
```diff
@@ -132,36 +131,17 @@ export class Banner extends Component {
...
+        <VaAlert
+          visible
+          full-width
+          closeable={showClose}
           onCloseAlert={onCloseAlert}
           status={type}
-        />
+        >
+          <h3 slot="headline">{title}</h3>
+          <div dangerouslySetInnerHTML={{ __html: content }} />
```
Comment (cvalarida) [inline src/components/Banner/Banner.jsx:143]: Mmm. :nauseated_face:  (I know it's the same as before.)

**Review by cvalarida — APPROVED** [review]

File: src/components/Banner/Banner.jsx (hunk 3 above)
Comment (bkjohnson) [reply src/components/Banner/Banner.jsx:143]: Yeah, I had to keep myself in scope for this :sweat_smile:

**Pull request merged**